src/lib/config.js

```diff
@@ -81,6 +81,17 @@ var getMongoDbPort = function() {
   return mongoPort;
 };
 
+/**
+ *  @returns boolean to define the RS as a configsvr or not. Default is false
+ */
+var isConfigRS = function() {
+  var configsvr = process.env.CONFIG_SVR || false;
+  if (configsvr) {
```

Comment:
This could be a little confusing because the string `"false"` will evaluate to `true`.